site2/website/blog/2021-02-03-Five-Minute-Interview-Elastic.md

```diff
@@ -0,0 +1,45 @@
+---
+author: Jonathan Ellis
+authorURL: https://twitter.com/spyced
+title: Five Minute Interview: Apache Pulsar with Elastic
+---
+
+This is the first in a series of quick interviews with members of the Apache Pulsar community to learn more about how Pulsar is being used in the wild.  Our guest today is [Ricardo Ferreira](https://twitter.com/riferrei).
+
+<!--truncate-->
+
+### What industry do you work in and what is your role?
+
+I work in the Technology/SaaS space. I work for Elastic, the company behind the Elastic Stack and Elastic Cloud. There, I am part of the community team focused on North America, working as a developer advocate.
+
+### What projects / use cases led to you adopting Pulsar?
+
+I have a background working with messaging and streaming data technologies. Whether working as Developer Advocate for Confluent (where my main focus was Kafka) or as a Solution Architect for Oracle I was constantly presented with opportunities to help customers to implement data pipelines using concepts of streaming and fast data, to replace batch oriented systems that are no longer relevant in today's economy where decisions must be taken instantly. When I was first introduced to Pulsar nearly 1.5 years ago I was impressed with its simplicity which led me to start digging deeper into the technology so I could expand my knowledge.
+
+Most recently I have been helping developers to adopt Pulsar in conjunction with Elasticsearch and Kibana — to allow them to implement end-2-end analytics solutions based on streaming data. Another use case that I have been helping developers with is on how to instrument their apps that write/read data streams to/from Pulsar to generate telemetry data compatible with the OpenTelemetry specification to Elastic APM.
```

Comment:
Do we have detailed info on this use case? or would the interviewee like to share more detailed info on the use case? sth like this https://www.infoq.com/articles/pulsar-customer-engagement-platform/